app/controllers/api/v1/sessions_controller.rb

```diff
@@ -9,13 +9,26 @@ def create
         self.resource = warden.authenticate!(auth_options)
         sign_in(resource_name, resource)
 
-        set_sso_response if sso_request?
-        @redirect_path ||= '/profile'
+        cookies[:token] = resource.token
```

Comment:
Later we should do something in the user model or sessions model for signins where we set the token.

Reply:
This didn't make much sense to me. Here's what I'm suggesting. anywhere we would normally call `sign_in` we now call some new thing `sign_in_cookies` which does the cookies part for us everywhere.

Reply:
That seems fair. How do I alter the `sign_in` method in that regard

Reply:
You could stick a small helper method in your ApiController that takes a user & additional optional args, if they all inherit from that.

I only see the `sign_in` method used 3x in the codebase, and never in the same context (different numbers of args). Anything that you add is probably going to have at least as many lines of code, and maybe even more. I don't think having it inline in the controller(s) is super horrible